types/types_test.go

```diff
@@ -127,14 +127,19 @@ func TestMemMarker_Count(t *testing.T) {
 	require.Equal(t, 1, countByState(AlertStateActive))
 	require.Equal(t, 1, countTotal())
 
-	// Insert a suppressed alert.
+	// Insert a suppressed alert by silence.
```

Comment:
Just a suggestion to make this easier to read:

```suggestion
	// Insert a silenced alert.
```